wrappers/wrapper.rb

```diff
@@ -367,6 +367,11 @@ def assert_no_vehicle_overall_duration_if_heuristic(vrp)
       vrp.vehicles.none?{ |vehicle| vehicle[:overall_duration] } || vrp.preprocessing_first_solution_strategy.to_a.first != 'periodic'
     end
 
+    def assert_no_overall_duration(vrp)
+      vrp.vehicles.none?{ |vehicle| vehicle[:overall_duration] } &&
+        vrp.relations.none?{ |relation| ['vehicle_group_duration', 'vehicle_group_duration_on_weeks', 'vehicle_group_duration_on_months'].include?(relation.type) }
```

Comment:
```suggestion
        vrp.relations.none?{ |relation| %w[vehicle_group_duration vehicle_group_duration_on_weeks vehicle_group_duration_on_months].include?(relation.type) }
```
I don't know if it is better in terms of perf but it is definitely more ruby syntax and it is easier to modify